### ISSUE TITLE
Pretrained weights for teacher model.

### ISSUE THREAD

**Issue by amyzxq** [harshm121/M3L#1]
hello! Thanks for the good research!
Did the initialization of the teacher model only use the parameters provided by pytorch? Or did you train a version on labeled data first?

**Comment by harshm121** [comment]
We started the teacher and the student from the same checkpoint (https://github.com/harshm121/M3L/blob/cc2aaaff326039fcb0de90ef4d0341fe343a6668/src/semi_supervised/meanteachermaskstudent.py#L241), without any warmup on the student or teacher (https://github.com/harshm121/M3L/blob/cc2aaaff326039fcb0de90ef4d0341fe343a6668/src/configs/sid_0.2_m3l.yml#L4).

**Comment by harshm121** [comment]
I hope this helps. Closing the issue.
